client/state/jetpack-connect/reducer.js

```diff
@@ -221,6 +222,12 @@ export function jetpackConnectAuthorize( state = {}, action ) {
 					bearerToken: action.data.bearer_token
 				}
 			);
+		case SITE_REQUEST_FAILURE:
+			const { client_id } = state.queryObject;
```

Comment:
@roccotripaldi do you mind following up with another PR to check if `queryObject` is available. The default state is `{}` so we may see the following. I notice this when I visit the purchases page, but I also have a lot of messed up test sites :)

![screen shot 2017-05-22 at 5 36 19 pm](https://cloud.githubusercontent.com/assets/1270189/26333806/72f3881a-3f15-11e7-9bad-a72b5c7576ab.png)